### DISCUSSION TITLE
Headless VueJS sample - how to make devServer open in a different browser?

### DISCUSSION THREAD

**Discussion by markgibbons25** [Sitecore/jss#1394] [Help]
Hi, I asked about this [here ](https://sitecore.stackexchange.com/questions/29896/headless-vuejs-sample-how-to-make-devserver-open-in-a-different-browser) but lack of response leads me to believe this isn't supported, so please could I request this as a feature or workaround?

I'm using the headless 18 vue sample, I'd like to have the devServer open a different browser, but the `open` config doesn't seem to make any difference (when putting that in `vue.config.js`. This is what I have tried - based on the documentation here: https://webpack.js.org/configuration/dev-server/

    vueConfig.devServer = {
      port: process.env.PORT || 3000,
      proxy:
        process.env.JSS_MODE === 'disconnected'
          ? `http://localhost:${process.env.PROXY_PORT || 3042}`
          : undefined,
      open: {
        app: {
          name: 'google-chrome',
          arguments: ['--incognito', '--new-window'],
        },
      },
    };

**Comment by CobyPear** [comment]
Hi @markgibbons25. What platform are you on? For windows it looks like the application name is `'chrome'`.

**Comment by markgibbons25** [comment]
Hi @CobyPear I'm on windows, I did try 'chrome' too as well as 'firefox' and 'edge'. I actually want it to work with [Blisk](https://blisk.io/).

**Comment by CobyPear** [comment]
Hmm perhaps [this is the solution](https://stackoverflow.com/a/34386559)? `open` doesn't seem to work on Windows. As far as specifying Blisk, you may need to set it to your default browser.
